docs/SPEC.md

```diff
@@ -1189,7 +1162,7 @@ All such values must have a corresponding builtin statement to declare the exist
 
 Example
 
-    builtin filter : (<-tables: [T], fn: (r: T) => bool) => [T]
+    builtin filter : (<-tables: stream[T], fn: (r: T) => bool) => stream[T]
```

Comment:
Is `stream` a keyword or reserved string here? I don't see this added to the grammar anywhere yet.

Reply:
`stream[]` is a type expression that is noted later in the SPEC.

Reply:
Yes, but it should be part of the BNF grammar as well.

Reply:
Nice catch, added to the grammar, now. However its done in such a way that it doesn't need to be a keyword.